one_fm/operations/doctype/process_task/process_task.py

```diff
@@ -264,7 +264,7 @@ def create_tasks_for(tasks_list):
 				task = process_task_doc.set_task_for_process_task()
 				if task:
 					process_task_doc.assign_employee_to_task(task)
-					process_task_doc.db_set('last_execution',datetime.now())
+					process_task_doc.db_set('last_execution',frappe.utils.now_datetime())
```

Comment:
We imported the now_datetime  Use the same here  https://github.com/ONE-F-M/one_fm/blob/cc40697e0a7e2a5980929f96b13172d3d78ac186/one_fm/operations/doctype/process_task/process_task.py#L17

Reply:
updated @pjamsheer